src/analysis/complex/real_deriv.lean

```diff
@@ -62,3 +62,46 @@ times_cont_diff_iff_times_cont_diff_at.2 $ λ x,
   h.times_cont_diff_at.real_of_complex
 
 end real_deriv_of_complex
+
+section conformality
+/-! ### Conformality of real-differentiable complex maps -/
+open complex continuous_linear_map
+
+variables
+
+/-- A real differentiable function of the complex plane into some complex normed space `E` is
+    conformal at a point `z` if it is holomorphic at that point -/
+lemma conformal_at_of_holomorph {E : Type*}
+  [normed_group E] [normed_space ℝ E] [normed_space ℂ E]
+  [is_scalar_tower ℝ ℂ E] {z : ℂ} {f : ℂ → E}
+  (hf' : fderiv ℝ f z ≠ 0) (h : differentiable_at ℂ f z) :
+  conformal_at f z :=
+begin
+  rw conformal_at_iff_is_conformal_map_fderiv,
+  rw (h.has_fderiv_at.restrict_scalars ℝ).fderiv at ⊢ hf',
+  apply is_conformal_map_complex_linear,
+  contrapose! hf' with w,
+  simp [w]
+end
+
+/-- A complex function is conformal if and only if the function is holomorphic or antiholomorphic
+    with a nonvanishing differential. -/
+lemma conformal_at_iff_holomorphic_or_antiholomorphic_at {f : ℂ → ℂ} {z : ℂ} :
```

Comment:
Again, to rephrase without `holomorphic_at` ... maybe
`conformal_at_iff_differentiable_at_or_differentiable_at_comp_conj`.